### PR TITLE
feat: add svelte-flicking

### PR DESCRIPTION
**Pull request by WoodNeck**

### CODE REVIEW

**Review by mixed — APPROVED** [review]
Welcome to svelte!

I have one question. Are the parts shown in the image below applied?
image

File: docs/src/demo/Sync.tsx [hunk 1]
```diff
@@ -181,6 +185,86 @@ export default {
...
+  let flicking2;
+
+  function update(flicking, progress) {
+    flicking.camera.lookAt(flicking.camera.range.min + flicking.camera.rangeDiff * progress);
+  };
+</script>
+
+<Flicking bind:this={flicking0} class="mb-4"
+  options={{ align: "prev", bound: true, bounce: 30 }}
+  on:move={e => {
```
Comment (daybrush) [inline docs/src/demo/Sync.tsx:203]: svelte CustomEvent should be accessed as detail.

https://svelte.dev/tutorial/component-events

File: packages/svelte-flicking/src/Flicking.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { withFlickingMethods } from "@egjs/flicking";
+
+import SvelteFlicking from "./flicking.svelte";
+
+export default (() => {
+  withFlickingMethods(SvelteFlicking, "vanillaFlicking");
```
Comment (daybrush) [inline packages/svelte-flicking/src/Flicking.js:6]: Shouldn't this method access the prototype?

**Review by daybrush — APPROVED** [review]
LGTM

File: docs/src/demo/Sync.tsx (hunk 1 above)
Comment (WoodNeck) [reply docs/src/demo/Sync.tsx:203]: Uhh, what do you mean?
If you're telling me to add some tutorials about how to use events in svelte, that's described in the other document.

File: packages/svelte-flicking/src/Flicking.js (hunk 2 above)
Comment (WoodNeck) [reply packages/svelte-flicking/src/Flicking.js:6]: You're right. I believe I've put prototype in that, wonder why it's gone.

**Comment by WoodNeck** [comment]
> Welcome to svelte!
> 
> I have one question. Are the parts shown in the image below applied?
> image

Of course!
image

File: docs/src/demo/Sync.tsx (hunk 1 above)
Comment (daybrush) [reply docs/src/demo/Sync.tsx:203]: The event we use is closer to ({ detail: e }) => {}.

Comment (WoodNeck) [reply docs/src/demo/Sync.tsx:203]: Oh, I got the point. I'll fix that 👍

**Pull request merged**